<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileDirAccessDemoClass</class>
 <widget class="QDialog" name="FileDirAccessDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileDirAccessDemo</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QToolBox" name="toolBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>175</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>175</width>
         <height>184</height>
        </rect>
       </property>
       <attribute name="label">
        <string>QCoreApplication类</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnGetDirInfo">
          <property name="text">
           <string>获取程序信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSetOrg">
          <property name="text">
           <string>设置机构名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnExit">
          <property name="text">
           <string>退出程序</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <attribute name="label">
        <string>QFile类</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btnFileOp">
          <property name="text">
           <string>复制、重命名、删除文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSetPermission">
          <property name="text">
           <string>设置文件的权限</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <attribute name="label">
        <string>QFileInfo类</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="btnGetFileInfo">
          <property name="text">
           <string>获取文件信息</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <attribute name="label">
        <string>QDir类</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="btnGetDirInformation">
          <property name="text">
           <string>获取目录信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDirOp">
          <property name="text">
           <string>文件目录操作</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLoopUpDir">
          <property name="text">
           <string>遍历文件目录</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_6">
       <attribute name="label">
        <string>QTemporaryDir和QTemporaryFile</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="btnTempDir">
          <property name="text">
           <string>临时目录操作</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnTempFile">
          <property name="text">
           <string>临时文件操作</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>175</width>
         <height>184</height>
        </rect>
       </property>
       <attribute name="label">
        <string>QFileSystemWatcher类</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="btnStartListen">
          <property name="text">
           <string>开启监听文件目录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnStopListen">
          <property name="text">
           <string>停止监听文件目录</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>491</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="btnOpenFile">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>打开文件</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnOpenDir">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>20</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>打开目录</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnClearText">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>20</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>清除文本框</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>文件</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtFile">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>431</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtDir">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>431</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>目录</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="ptInfo">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>160</y>
         <width>431</width>
         <height>161</height>
        </rect>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="overwriteMode">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>信息</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FileDirAccessDemo.qrc"/>
 </resources>
 <connections/>
</ui>
